pulp/__init__.py

```diff
@@ -36,6 +36,7 @@
 from .apis import *
 from .utilities import *
 from .constants import *
+from .tests import *
```

Comment:
I think I prefer only exposing `pulpTestAll` just to be safe so:

`from .tests import pulpTestAll`

Reply:
You are right. It is better to expose less members of package.